src/backends/plonky2/circuits/mainpod.rs

```diff
@@ -1752,24 +1734,29 @@ mod tests {
         let prev_statements_target: Vec<_> = (0..prev_statements.len())
             .map(|_| builder.add_virtual_statement(&params))
             .collect();
+
         let merkle_proofs_target: Vec<_> = merkle_proofs
             .iter()
             .map(|_| {
-                let mt_proof = MerkleClaimAndProofTarget::new_virtual(
-                    params.max_depth_mt_containers,
-                    &mut builder,
-                );
-                verify_merkle_proof_circuit(&mut builder, &mt_proof);
```

Comment:
Removed this verification because it's also happening later via `build_operation_aux_table_circuit`